docs/core/getting_started/installation.md

```diff
@@ -53,17 +54,25 @@ Examples of extra packages include:
 
 ## Running the local server and UI
 
-In addition to the [Prefect Cloud](https://www.prefect.io/cloud) platform, Prefect includes an open-source server and UI for orchestrating and managing flows. The local server stores flow metadata in a Postgres database and exposes a GraphQL API.
+Prefect includes an open-source server and UI for orchestrating and managing flows. The local server stores flow metadata in a Postgres database and exposes a GraphQL API. The local server requires [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/) to be running. If you've installed [Docker Desktop](https://www.docker.com/products/docker-desktop), you've got both of these.
```

Comment:
> requires Docker and Docker-Compose to be running

Slight quibble - Docker-Compose is a CLI tool, not something that needs to be running in the background (but `docker` does need to be running). Not sure how to tweak the language to better clarify that. Perhaps replace "to be running" with "to be installed"?

Reply:
Installed is a much better word in this case. I'll commit an update and push it in. Thanks!